examples/model_selection/plot_grid_search_stats.py

```diff
@@ -0,0 +1,573 @@
+      f"{model_scores.index[0]}: {1 - better_prob:.3f}")
+
+# %%
+# In contrast with the frequentist approach, we can compute the probability
+# that one model is better than the other.
+#
+# Note that we obtained similar results as those in the frequentist approach.
+# Given our choice of priors, we are essentially performing the same
+# computations, but we are allowed to make different assertions.
+
+# %%
+# Region of Practical Equivalence
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# Sometimes we are interested in determining the probabilities that our models
+# have an equivalent performance, where "equivalent" is defined in a practical
+# way. A naive approach [4]_ would be to define estimators as practically
+# equivalent when they differ by less than 1% in their accuracy. But we could
+# also define this practical equivalence taking into account the problem we are
+# trying to solve. For example, a difference of 5% in accuracy would mean an
+# increase of $1000 in sales, and we consider any quantity above that as
+# relevant for our business.
+#
+# In this example we are going to define the
+# Region of Practical Equivalence (ROPE) to be :math:`[-0.01, 0.01]`. That is,
+# we will consider two models as practically equivelent if they differ by less
+# than 1% in their performance.
+#
+# To compute the probabilities of the classifiers being practically equivalent,
+# we calculate the area under the curve of the posterior over the ROPE
+# interval:
+
+rope_interval = [-0.01, 0.01]
+rope_prob = t_post.cdf(rope_interval[1]) - t_post.cdf(rope_interval[0])
+
+print(f"Probability of {model_scores.index[0]} and {model_scores.index[1]} "
+      f"being practically equivalent: {rope_prob:.3f}")
+
+# %%
+# We can plot how the posterior is distributed over the ROPE interval:
+
+x_rope = np.linspace(rope_interval[0], rope_interval[1], 100)
+
+plt.plot(x, t_post.pdf(x))
+plt.xticks(np.arange(-0.04, 0.06, 0.01))
+plt.vlines([-0.01, 0.01], ymin=0, ymax=(np.max(t_post.pdf(x)) + 1))
+plt.fill_between(x_rope, t_post.pdf(x_rope), 0, facecolor='blue', alpha=.2)
+plt.ylabel("Probability density")
+plt.xlabel("Mean difference")
+plt.title("Posterior distribution under the ROPE")
+plt.show()
+
+# %%
+# As suggested in [4]_, we can further interpret these probabilities using the
+# same criteria as the frequentist approach: Is the probability of falling
+# inside the ROPE bigger than 95% (alpha value of 5%)?  In that case we can
+# conclude that both models are practically equivalent.
+
+# %%
+# The Bayesian estimation approach also allows us to compute how uncertain we
+# are about our estimation of the difference. This can be calculated using
+# credible intervals which show the range of values that the mean
+# difference in performance between models can take with certain probability.
+# For example, a credible interval of 50% will tell us that there is a 50% of
+# probability that the true (mean) difference of performance between models is
+# between x and y values.
```

Comment:
```suggestion
# are about our estimation of the difference. This can be calculated using
# credible intervals. For a given probability, they show the range of values
# that the estimated quantity, in our case the mean difference in performance,
# can take. For example, a 50% credible interval [x, y] tells us that there
# is a 50% probability that the true (mean) difference of performance between
# models is between x and y.
```

Reply:
thanks! this looks much better